intake_sql/intake_sql.py

```diff
@@ -39,8 +39,8 @@ def __init__(self, uri, sql_expr, sql_kwargs={}, metadata={}):
 
     def _load(self):
         import pandas as pd
-        self._dataframe = pd.read_sql(self._sql_expr, self._uri,
-                                      **self._sql_kwargs)
+        loader = pd.read_sql_table if self._sql_kwargs.get("schema") else pd.read_sql
```

Comment:
`schema` is a valid kwarg in the `read_sql_table` variant of this function, but not the `read_sql_query` variant, hence this somewhat awkward construction.